.github/scripts/get_system_info.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+#  Copyright (C) 2023 Intel Corporation
+#  Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+#  See LICENSE.TXT
+#  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# get_system_info.sh - Script for printing system info
+
+function system_info {
+	echo "********** system_info **********"
+	cat /etc/os-release | grep -oP "PRETTY_NAME=\K.*"
+	echo "**********SYCL-LS**********"
+	source /opt/intel/oneapi/setvars.sh
+	sycl-ls
+	echo "*************** installed-packages ***************"
+	# Instructions below will return some minor errors, as they are dependent on the Linux distribution.
```

Comment:
you can move this section to the bottom; in general we should have the most significant (for us) info on the top, I think

Reply:
done